interface/main/tabs/main.php

```diff
@@ -148,65 +174,18 @@ function isEncounterLocked(encounterId) {
             return false;
             <?php } ?>
         }
-
-        // some globals to access using top.variable
-        window.name = "main";
-        // note that 'let' or 'const' does not allow global scope here.
-        // only use var
-        var userDebug = <?php echo js_escape($GLOBALS['user_debug']); ?>;
-        var webroot_url = <?php echo js_escape($web_root); ?>;
-        var jsLanguageDirection = <?php echo js_escape($_SESSION['language_direction']); ?>;
-        var jsGlobals = {};
     </script>
 
     <?php Header::setupHeader(['knockout', 'tabs-theme', 'i18next']); ?>
 
     <link rel="shortcut icon" href="<?php echo $GLOBALS['images_static_relative']; ?>/favicon.ico" />
 
-    <script type="text/javascript" src="js/custom_bindings.js?v=<?php echo $v_js_includes; ?>"></script>
-    <script type="text/javascript" src="js/user_data_view_model.js?v=<?php echo $v_js_includes; ?>"></script>
-    <script type="text/javascript" src="js/patient_data_view_model.js?v=<?php echo $v_js_includes; ?>"></script>
-    <script type="text/javascript" src="js/therapy_group_data_view_model.js?v=<?php echo $v_js_includes; ?>"></script>
+    <script src="js/custom_bindings.js?v=<?php echo $v_js_includes; ?>"></script>
+    <script src="js/user_data_view_model.js?v=<?php echo $v_js_includes; ?>"></script>
+    <script src="js/patient_data_view_model.js?v=<?php echo $v_js_includes; ?>"></script>
+    <script src="js/therapy_group_data_view_model.js?v=<?php echo $v_js_includes; ?>"></script>
```

Comment:
looks lines 179-186 are duplicated below :)

Reply:
sure is! hmm